### PR TITLE
Enhancement: Run php-cs-fixer on Travis

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] removes `squizlabs/php_codesniffer`
* [x] adds `fabpot/php-cs-fixer`
* [x] adds a configuration for `php-cs-fixer`
* [x] performs a dry-run of `php-cs-fixer` on Travis
* [x] performs an actual run of `php-cs-fixer`

Replaces #20.

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        'object_operator',
+        'operators_spaces',
+        'ordered_use',
+        // 'phpdoc_indent',
+        // 'phpdoc_params',
+        'remove_leading_slash_use',
+        'remove_lines_between_uses',
+        'return',
+        'single_array_no_trailing_comma',
+        // 'short_array_syntax',
```
Comment (localheinz) [inline .php_cs]: See https://github.com/EvanDotPro/EdpGithub/blob/master/composer.json#L26.

**Comment by gianarb** [comment]
@localheinz in this moment this is very difficult to merge! :cold_sweat:

**Comment by localheinz** [comment]
@gianarb 

How about this, now? I rebased and ran the fixer again.

**Comment by gianarb** [comment]
Ook! :) I merge it after your rebase!
Thanks

**Comment by GrahamCampbell** [comment]
Why not use https://styleci.io? It separates this from the actual tests you can see which bit failed straight away.

**Comment by gianarb** [comment]
Hi! @GrahamCampbell this is the evolution of FabBot?! :)
In my opinion this is a good idea! The CsFix is not a total failure for me.. But a simple status to underline it is the solution!

**Comment by GrahamCampbell** [comment]
> this is the evolution of FabBot?! :)

This is totally separate from fabot. Fabot is closed source, so I wrote StyleCI and made it open source. :)

**Comment by GrahamCampbell** [comment]
And am providing a free hosted service. :)

**Comment by gianarb** [comment]
Beautiful! :) Thanks for your work!

**Comment by localheinz** [comment]
@GrahamCampbell 

TANSTAAFL

**Pull request merged**